src/aap_eda/settings/default.py

```diff
@@ -806,8 +797,13 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     f"port={DATABASES['default']['PORT']} "
     f"dbname={DATABASES['default']['NAME']} "
     f"user={DATABASES['default']['USER']} "
-    f"password={DATABASES['default']['PASSWORD']}"
+    f"password={DATABASES['default']['PASSWORD']} "
+    f"sslmode={DATABASES['default']['OPTIONS']['sslmode']} "
+    f"sslcert={DATABASES['default']['OPTIONS']['sslcert']} "
+    f"sslkey={DATABASES['default']['OPTIONS']['sslkey']} "
```

Comment:
The optional params must be dynamic or we must make sure first they already exists, either in a different function or in `_get_databases_settings`
because DATABASES can be arbitrarily provided through the settings.yaml, otherwise we would raise KeyError here.